client/asset/btc/btc.go

```diff
@@ -1935,23 +1939,21 @@ func (btc *baseWallet) estimateSwap(lots, lotSize, feeSuggestion, maxFeeRate uin
 	} else {
 		estLowFunds += dexbtc.P2SHOutputSize * (lots - 1) * bumpedNetRate
 	}
-
 	estLowFees := estLowFunds - val
 
 	// Math for split transactions is a little different.
 	if trySplit {
-		_, extraMaxFees := btc.splitBaggageFees(bumpedMaxRate)
+		_, splitMaxFees := btc.splitBaggageFees(bumpedMaxRate)
 		_, splitFees := btc.splitBaggageFees(bumpedNetRate)
-		locked := val + maxFees + extraMaxFees
-
-		if avail >= reqFunds+extraMaxFees {
+		reqTotal := reqFunds + splitMaxFees // ~ rather than actually fund()ing again
+		if reqTotal <= sum {
 			return &asset.SwapEstimate{
 				Lots:               lots,
 				Value:              val,
-				MaxFees:            maxFees + extraMaxFees,
+				MaxFees:            maxFees + splitMaxFees,
 				RealisticBestCase:  estLowFees + splitFees,
 				RealisticWorstCase: estHighFees + splitFees,
-			}, true, locked, nil
+			}, true, reqFunds, nil // requires reqTotal, but locks reqFunds in the split output
```

Comment:
This was another bug, but not very important.  The amount locked by the funding coins for an order funded by a split transaction is precisely the value of the output of the split transaction i.e. reqFunds.  The "extra" fees are just burned in making the split transaction, so it's not correct to also call them locked since they are included in the various fee estimates as well.

The tests didn't check for this because we don't test estimateSwap directly, and this locked amount goes nowhere except the test messages in the generated config opt for the UI.  The `checkSwapEstimate` helper hand an unused `locked` arg and I can see why it was unused.  I'm not planning to add new tests for this however.